SU-SWS/stanford_profile
# D8CORE-855 Move toolbar user tab to far right

# READY FOR REVIEW

# Summary
- Moved the "User" tab to the right of the toolbar.

# Need Review By (Date)
- 12/11

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear cache
1. verify the user tab in the toolbar is at the right.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
